.github/workflows/cmake.yml

```diff
@@ -207,7 +210,13 @@ jobs:
           -D CMAKE_BUILD_TYPE=${{ matrix.build.type }}
           -D CMAKE_C_COMPILER=gcc-11 -D CMAKE_CXX_COMPILER=g++-11
       - name: Build
-        run: cmake --build build --target install
+        run: cmake --build build
+      - name: Generate package
+        run: cmake --install build --prefix ./build/artifacts ${{ matrix.build.strip }}
+      - uses: actions/upload-artifact@v2
+        with:
+          name: '86Box${{ matrix.build.slug }}-Linux-x86_64-gha${{ github.run_number }}'
+          path: build/artifacts/**
```

Comment:
Nitpick: I would avoid creating any Linux artifacts that are not self-contained and statically linked with all dependencies, otherwise the binary is useless on pretty much anything that is not the host distro.

Reply:
Yeah, I'm just cleaning stuff up, I recall the conversation about static linking and fully agree with you too. But that's for another day, unless you want to beat me to it.